app/src/test/java/org/mozilla/fenix/ext/ImageButtonTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.fenix.ext
+
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import org.mozilla.fenix.TestApplication
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import io.mockk.mockk
+import io.mockk.verify
+import android.widget.ImageButton
+
+@ObsoleteCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
+
+class ImageButtonTest {
```

Comment:
nit: there shouldn't be newline of whitespace between the class and the annotations